src/requirements/data/colleges/asFA2020.ts

```diff
@@ -100,3 +102,13 @@ const casFA2020Requirements: readonly CollegeOrMajorRequirement[] = [
 ];
 
 export default casFA2020Requirements;
+
+export const casAdvisors: AdvisorGroup = {
+  advisors: [
+    {
+      name: 'Jayla Greene',
+      email: 'jng65@cornell.edu',
+      checker: lastNameRange('https://as.cornell.edu/advising', ''),
```

Comment:
Is this supposed to be a source?

Reply:
Yes, most likely! @zachary0kent I think you might be off the hook for this… I think @michaelfarkouh put this in the wrong column in the spreadsheet :)